context/app/static/js/shared-styles/buttons/index.tsx

```diff
@@ -69,7 +69,7 @@ function TooltipToggleButton({
 
   return (
     <Tooltip title={tooltipTitle}>
-      <span>
+      <span role="tooltip">
```

Comment:
I'm not sure this is the most appropriate role for the wrapper of the tooltip anchor, since the MDN example has the `tooltip` role on the div with the tooltip contents, rather than the anchor: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/tooltip_role

Maybe we could use `useId` to generate a unique tooltip ID, pass it to `Tooltip` as the `id`, and use the `aria-describedby` prop on the span or button instead?